.github/workflows/preview-build.yml

```diff
@@ -1,7 +1,7 @@
 name: preview-build
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
IIUC, this will not let you use `write` permissions scopes. Is that the expected behaviour?